CASESTUDY.md

```diff
@@ -23,7 +23,7 @@ Here's a detailed scenario that encapsulates the token transformation and transf
 
 This scenario illustrates the conversion of token states from non-fungible (ERC721) to fungible (ERC20) and their transfer between parties, showcasing the flexibility and liquidity options provided by this innovative token standard.
 
-We can conceptualise the table based on the scenario provided, where Alice sends an ERC721 token to Bob, and then Bob transforms this token to send 0.5 of an ERC20 token to Charle. The `balanceOf()` function's return values for Alice, Bob, and John will be shown in the context of before and after the transactions:
+We can conceptualise the table based on the scenario provided, where Alice sends an ERC721 token to Bob, and then Bob transforms this token to send 0.5 of an ERC20 token to Charle. The `balanceOf()` function's return values for Alice, Bob, and Charle will be shown in the context of before and after the transactions:
```

Comment:
@futantan "Charlie"

Reply:
nice catch! but all other places use `Charle`, I update `John` to `Charle` to keep consistency.
Not sure if we need to change all 25 `Charle` to `Charlie`